styles/globals.css

```diff
@@ -14,7 +14,125 @@ a {
 * {
   box-sizing: border-box;
 }
+@import url("https://rsms.me/inter/inter.css");
 
+:root {
+  --color-gray: #737888;
+  --color-lighter-gray: #e3e5ed;
+  --color-light-gray: #f7f7fa;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+html {
+  font-family: "Inter", sans-serif;
+  font-size: 14px;
+  box-sizing: border-box;
+}
+
+@supports (font-variation-settings: normal) {
+  html {
+    font-family: "Inter var", sans-serif;
+  }
+}
+
+body {
+  margin: 0;
+}
+
+h1 {
```

Comment:
wow! so much css. the point of tailwind is not to write much css, especially not the stuff that tailwind supports out of the box. any reason why you decided to stick to raw css over tailwind?

Reply:
No reason actually. I'm not used to the tailwind. Definitely, I'm gonna try it out!